src/_guides/language/analysis-options.md

```diff
@@ -381,6 +381,44 @@ linter:
   You can, however, use the other syntax for rules in an included file.
 {{site.alert.end}}
 
+## Enabling analyzer plugins (experimental)
```

Comment:
Things have changed. While we are still not comfortable with the current support we need to document the experimental feature because users are confused and filing bugs when they hit corner cases such as only one plugin per package.

Reply:
Thanks for the extra context, definitely makes sense!